source/index.html.md

```diff
@@ -19,221 +17,620 @@ search: true
-    "name": "Fluffums",
-    "breed": "calico",
-    "fluffiness": 6,
-    "cuteness": 7
-  },
-  {
-    "id": 2,
-    "name": "Max",
-    "breed": "unknown",
-    "fluffiness": 5,
-    "cuteness": 10
-  }
-]
+{
+  "from": "0x87265a62c60247f862b9149423061b36b460f4bb",
+  "nonce": "0x14",
+  "gasPrice": "0x0",
+  "gasLimit": "0xf4240",
+  "to": "0x883ed48b3210082cf82fb92ce81f0b17bdec4f81",
+  "value": "0x0",
+  "data": "0xa9059cbb000000000000000000000000b99c958777f024bc4ce992b2a0efb2f1f50a4dcf000000000000000000000000000000000000000000000000000000000000000a",
+  "chainId": 17224
+}
 ```
 
-This endpoint retrieves all kittens.
+This endpoint returns a raw transaction (unsigned) which uses the nonce based on the blockchain `transactionCount` for that address.
+
+
+<aside class="warning"> If an SDK is available for your programming language, we highly recommend you use the `Wallet` class to construct valid transactions and send them instead of this function, since it takes care of transaction construction and signing for you. </aside>
+
 
 ### HTTP Request
 
-`GET http://example.com/api/kittens`
+`GET https://api.qiibee.com/transactions/raw`
 
 ### Query Parameters
 
 Parameter | Default | Description
 --------- | ------- | -----------
-include_cats | false | If set to true, the result will also include cats.
-available | true | If set to false, the result will include kittens that have already been adopted.
+from | None | Address of the sender
+to | None | Receiver address of the loyalty tokens
+contractAddress | None | Loyalty Token contract address. (Use either the symbol or the contractAddress, not both)
+symbol | None | Loyalty Token symbol. (Use either the symbol or the contractAddress, not both)
+transferAmount | None | Amount of loyalty tokens being sent (in wei)
 
-<aside class="success">
-Remember — a happy kitten is an authenticated kitten!
-</aside>
+## Post signed transaction
 
-## Get a Specific Kitten
 
-```ruby
-require 'kittn'
+```python
+import qbsdk
+
+api = qbsdk.Api('my_secret_api')
 
-api = Kittn::APIClient.authorize!('meowmeowmeow')
-api.kittens.get(2)
+signed_tx_string = '0xf8aa8305ee6e80830f424094f80c1e40d50f383330a886f03f147f49342ad7c780b844a9059cbb00000000000000000000000087265a62c60247f862b9149423061b36b460f4bb000000000000000000000000000000000000000000000000000000000000000a8286b5a0e8a236e4b57e79ee0eec88d8bc7a7ef1c44a1d181bad366621e72c341490e550a0683ce6443acc4384ea2b18fcaba416b8f7379963b6a0c596cea423cade502eb3'
+
+api.post_transaction(signed_tx_string)
 ```
 
+```shell
+curl -d '{"data": "0xf8aa8305ee6e80830f424094f80c1e40d50f383330a886f03f147f49342ad7c780b844a9059cbb00000000000000000000000087265a62c60247f862b9149423061b36b460f4bb000000000000000000000000000000000000000000000000000000000000000a8286b5a0e8a236e4b57e79ee0eec88d8bc7a7ef1c44a1d181bad366621e72c341490e550a0683ce6443acc4384ea2b18fcaba416b8f7379963b6a0c596cea423cade502eb3" }' -H "Content-Type: application/json" -X POST  https://apitesting.qiibee.com/transactions
```

Comment:
why are we mixing api and apitesting in the examples?

Reply:
paste error